ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -171,6 +171,9 @@ router bgp {{ router_bgp.as }}
 {%         if peer_group.ebgp_multihop is arista.avd.defined %}
    neighbor {{ peer_group.name }} ebgp-multihop {{ peer_group.ebgp_multihop }}
 {%         endif %}
+{%         if peer_group.ttl_maximum_hops is arista.avd.defined %}
+   neighbor {{ peer_group.ip_address }} ttl maximum-hops {{ peer_group.ttl_maximum_hops }}
```

Comment:
neighbor {{ peer_group.name }} ttl maximum-hops {{ peer_group.ttl_maximum_hops }}

Reply:
Yes I pushed out the fix in the last commit

Reply:
confirmed